src/pages/course/Course.page.tsx

```diff
@@ -2,6 +2,8 @@ import { FC } from 'react';
 
 import { SingleCourse } from '../../modules/single-course/SingleCourse.module';
 
-export const Course: FC = () => {
+const Course: FC = () => {
```

Comment:
Like in the App.tsx, you can write arrow functions without curly brackets and `return` operator.